README.md

```diff
@@ -41,3 +41,46 @@ spec:
 # Screenshots
 
 ![picture](screenshot.png)
+
+# JSONL reader
+
+## API
+
+This functionality needs `th2-json-stream-provider 0.0.7 (or newer)` backend component to function. It should be accessible at `{rpt-viewer-path}/json-stream-provider*`
+
+## Features
```

Comment:
Please add a separate topic with correct examples of each file:
* simple notebook with all kinds of parameters
* result with all kinds for key words
* search file
* parameter values

Reply:
Isn't this examples should be on provider read me?

Reply:
So we have tree entities:
* viewer which can work with specific file formats
* notebook which one nowhere described and can have any code
* js-p which handle request, pass some path to parameters, executes notebook and return result in any JSONL format.

I think detail for files are fully denuded on viewer